src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -215,6 +225,31 @@ typedef struct uct_ib_mlx5_devx_umem {
     size_t                   size;
 } uct_ib_mlx5_devx_umem_t;
 #endif
+typedef struct uct_ib_mlx5_mini_cqe8 {
+    union {
+        uint32_t rx_hash_result;
+        struct {
+            union {
+                uint16_t checksum;
+                uint16_t flow_tag_high;
+                struct {
+                    uint8_t reserved1;
+                    uint8_t hdr_type;
+                };
+            };
+            uint16_t stride_idx;
+        };
+        struct {
+            uint16_t wqe_counter;
+            uint8_t  s_wqe_opcode;
+            uint8_t  reserved2;
+        };
+    };
+    union {
+        uint32_t byte_cnt_flow;
+        uint32_t byte_cnt;
+    };
+} uct_ib_mlx5_mini_cqe8_t;
```

Comment:
This structure contains a lot of fields that aren't used in the code but described in the documentation. Should I cleanup the unused field or leave them as now for the future purposes?

Reply:
yes, cleanp unused